benchmark_config_directory: $(find eurobench_benchmark_core)/config/benchmark_config
benchmark_output_directory: "~/eurobench_output"

robot_names:
  - "1"
  - "2"

excluded_topics:
  - /bmcore/state

# Seconds until benchmark start time
benchmark_countdown: 5
